Rakefile

```diff
@@ -25,3 +25,8 @@ desc 'run rspec via windows'
 task :win do
   system('rspec --tag win')
 end
+
+desc 'run immanent spec'
+task :immanent_spec do
```

Comment:
Are we sure that immanent is a good word? I think I don't even know the meaning of this word and if we google `immanent software tests` - there is no results except some company named like that

Isn't it just integration tests or unit tests?

Reply:
Yes, there are regular problems with the names

Theses and problems

1. We want to run unit and integration test on the same pipeline in github actions to optimize resources

2. We have allocated a separate folder for unit tests in order to share them with the tests on the builder

3. rake runs tests from spec by default

To summarize, in order to run unit and integration tests in the docker container in one pipe, the easiest way is to allocate them into one folder with the spec_helper.rb entry point

And the term immanens is my attempt to be clever. unit and integration tests are different entities, but they are united by the direction of testing. Namely, the logic of the project. Those are the immanent properties of the project.

Reply:
Up to you with this naming, but for many-many years of learning English in format and official way - this is my first time seeing this word...

I'm not sure that even english natives know it :)

Reply:
Yes, this term is rooted in Latin and is used more often by philosophers than by engineers.

But there is a small problem in which category to allocate unit and integration tests and separate them from the tests on builder)

Reply:
Maybe

`project spec` or `system spec`?

Or already spit and split in the actinos run unit and integration run